grammar.js

```diff
@@ -288,9 +288,31 @@ module.exports = grammar({
         optional($._paperTbl),
         optional($._formatingProperties),
         optional($.parTbl),
-        $.textUnit
+        choice($._fieldArea, $.textUnit)
       ),
 
+    _fieldArea: ($) => seq("{\\field", $._fieldinst, $._fieldslt, "}"),
+
+    _fieldinst: ($) => seq("{\\*", "\\fldinst", $._hyperlinkUnit, "}"),
+
+    _fieldslt: ($) => seq("{\\fldrslt", " ", $.textUnit, "}"),
+
+    _hyperlinkUnit: ($) => seq('{HYPERLINK "', $._static_URL_literal, '"}'),
+
+    _textUnitParameters: ($) =>
+      seq(repeat1(seq(repeat1($._textUnit_config), " "))),
+
+    _something: ($) =>
```

Comment:
Shall we rename this to something more meaningful?

Reply:
Thanks, updated!